<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Teclado_numericoClass</class>
 <widget class="QMainWindow" name="Teclado_numericoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Teclado_numerico</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="Teclado">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>220</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="Teclado_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: white;
border: 2px solid #5E5E5E;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy:-0.1,
radius: 1.3, stop: 0 rgba(198,198,198, 255), stop: 1 rgba(50, 50,50,50));
}

QPushButton:pressed{
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.1,
radius: 1.35, stop: 0 rgba(50, 50,50,50), stop: 1 rgba(198, 198,198, 255));
}</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Teclado_8">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: white;
border: 2px solid #5E5E5E;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy:-0.1,
radius: 1.3, stop: 0 rgba(198,198,198, 255), stop: 1 rgba(50, 50,50,50));
}

QPushButton:pressed{
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.1,
radius: 1.35, stop: 0 rgba(50, 50,50,50), stop: 1 rgba(198, 198,198, 255));
}</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Teclado_9">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: white;
border: 2px solid #5E5E5E;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy:-0.1,
radius: 1.3, stop: 0 rgba(198,198,198, 255), stop: 1 rgba(50, 50,50,50));
}

QPushButton:pressed{
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.1,
radius: 1.35, stop: 0 rgba(50, 50,50,50), stop: 1 rgba(198, 198,198, 255));
}</string>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Teclado_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: white;
border: 2px solid #5E5E5E;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy:-0.1,
radius: 1.3, stop: 0 rgba(198,198,198, 255), stop: 1 rgba(50, 50,50,50));
}

QPushButton:pressed{
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.1,
radius: 1.35, stop: 0 rgba(50, 50,50,50), stop: 1 rgba(198, 198,198, 255));
}</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Teclado_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: white;
border: 2px solid #5E5E5E;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy:-0.1,
radius: 1.3, stop: 0 rgba(198,198,198, 255), stop: 1 rgba(50, 50,50,50));
}

QPushButton:pressed{
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.1,
radius: 1.35, stop: 0 rgba(50, 50,50,50), stop: 1 rgba(198, 198,198, 255));
}</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Teclado_6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: white;
border: 2px solid #5E5E5E;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy:-0.1,
radius: 1.3, stop: 0 rgba(198,198,198, 255), stop: 1 rgba(50, 50,50,50));
}

QPushButton:pressed{
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.1,
radius: 1.35, stop: 0 rgba(50, 50,50,50), stop: 1 rgba(198, 198,198, 255));
}</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Teclado_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: white;
border: 2px solid #5E5E5E;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy:-0.1,
radius: 1.3, stop: 0 rgba(198,198,198, 255), stop: 1 rgba(50, 50,50,50));
}

QPushButton:pressed{
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.1,
radius: 1.35, stop: 0 rgba(50, 50,50,50), stop: 1 rgba(198, 198,198, 255));
}</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Teclado_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: white;
border: 2px solid #5E5E5E;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy:-0.1,
radius: 1.3, stop: 0 rgba(198,198,198, 255), stop: 1 rgba(50, 50,50,50));
}

QPushButton:pressed{
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.1,
radius: 1.35, stop: 0 rgba(50, 50,50,50), stop: 1 rgba(198, 198,198, 255));
}</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Teclado_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: white;
border: 2px solid #5E5E5E;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy:-0.1,
radius: 1.3, stop: 0 rgba(198,198,198, 255), stop: 1 rgba(50, 50,50,50));
}

QPushButton:pressed{
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.1,
radius: 1.35, stop: 0 rgba(50, 50,50,50), stop: 1 rgba(198, 198,198, 255));
}</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Teclado_coma">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: white;
border: 2px solid #5E5E5E;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy:-0.1,
radius: 1.3, stop: 0 rgba(198,198,198, 255), stop: 1 rgba(50, 50,50,50));
}

QPushButton:pressed{
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.1,
radius: 1.35, stop: 0 rgba(50, 50,50,50), stop: 1 rgba(198, 198,198, 255));
}</string>
     </property>
     <property name="text">
      <string>,</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Teclado_0">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: white;
border: 2px solid #5E5E5E;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy:-0.1,
radius: 1.3, stop: 0 rgba(198,198,198, 255), stop: 1 rgba(50, 50,50,50));
}

QPushButton:pressed{
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.1,
radius: 1.35, stop: 0 rgba(50, 50,50,50), stop: 1 rgba(198, 198,198, 255));
}</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Teclado_cl">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>220</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: white;
border: 2px solid #5E5E5E;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy:-0.1,
radius: 1.3, stop: 0 rgba(198,198,198, 255), stop: 1 rgba(50, 50,50,50));
}

QPushButton:pressed{
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.1,
radius: 1.35, stop: 0 rgba(50, 50,50,50), stop: 1 rgba(198, 198,198, 255));
}</string>
     </property>
     <property name="text">
      <string>Cl</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Teclado_ok">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>90</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>17</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: white;
border: 2px solid #5E5E5E;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy:-0.1,
radius: 1.3, stop: 0 rgba(198,198,198, 255), stop: 1 rgba(50, 50,50,50));
}

QPushButton:pressed{
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.1,
radius: 1.35, stop: 0 rgba(50, 50,50,50), stop: 1 rgba(198, 198,198, 255));
}</string>
     </property>
     <property name="text">
      <string>Enter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Teclado_cancel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>290</y>
       <width>90</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>17</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: white;
border: 2px solid #5E5E5E;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy:-0.1,
radius: 1.3, stop: 0 rgba(198,198,198, 255), stop: 1 rgba(50, 50,50,50));
}

QPushButton:pressed{
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.1,
radius: 1.35, stop: 0 rgba(50, 50,50,50), stop: 1 rgba(198, 198,198, 255));
}</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Boton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="teclado_numerico.qrc"/>
 </resources>
 <connections/>
</ui>
